samples/django/django_user/myoauth/urls.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2016 Google Inc.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from django.conf.urls import include, url
+from django.contrib import admin
+import django.contrib.auth.views
```

Comment:
Please conform to [Google style](https://google.github.io/styleguide/pyguide.html) for two reasons: (1) this is Google-authored code, and (2) "this code demonstrates use of library <x>, therefore it should be implemented in the style in which library <x> was implemented" strikes me as inappropriate coupling, nonsense, and not something that I find at all persuasive. I feel a little bit negotiable if you want to push back, though?